source/extensions/filters/network/dubbo_proxy/router/BUILD

```diff
@@ -32,3 +32,40 @@ envoy_cc_library(
         "@envoy_api//envoy/config/filter/network/dubbo_proxy/v2alpha1:dubbo_proxy_cc",
     ],
 )
+
+envoy_cc_library(
+    name = "config",
+    srcs = ["config.cc"],
+    hdrs = ["config.h"],
+    deps = [
+        ":router_lib",
+        ":router_matcher",
```

Comment:
route_matcher?

Reply:
removed.

Reply:
replace router_matcher to route_matcher?

Reply:
The router_matcher dependency has been removed and the router_matcher library name renamed to route_matcher.